research/gaia/gremlin/compiler/gremlin-server-plugin/src/main/java/com/alibaba/graphscope/gaia/plan/translator/builder/PlanConfig.java

```diff
@@ -19,5 +19,5 @@
     public static String QUERY_ID = "query_id";
     public static String TAG_ID_MAKER = "tag_id_maker";
     public static String QUERY_CONFIG = "query_config";
-    public static String SNAPSHOT_ID = "snapshot_id";
+    public static String SNAPSHOT_ID = "SID";
```

Comment:
Add a comment: "This snapshot id is aligned with the runtime @(bingqing) to represent the extra parameter of snapshot id while accessing MaxGraphStore. "